src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -1021,18 +1024,21 @@ class Program
                     static void Main()
                     {
                         var x = F([1, null]);
+                        x.Report(includeType: true);
                         int?[] y = [null, 2];
                         var z = F([..y]);
+                        z.Report(includeType: true);
                     }
```

Comment:
>} [](http://example.com/codeflow?start=20&length=1)

This looks like the only test that hits the spread element case in `OverloadResolution.BetterCollectionExpressionConversion()`. Let's consider also testing cases where there are more than element. Perhaps:
```
F([3, ..y]).Report(includeType: true);
F([..y, 4]).Report(includeType: true);
int[] w = [5, 6, 7];
F([..y, ..w]).Report(includeType: true);
F([..w, ..y]).Report(includeType: true);
```